xbmc/cores/IPlayer.h

```diff
@@ -159,9 +159,6 @@ enum EINTERLACEMETHOD
   VS_INTERLACEMETHOD_VDPAU_TEMPORAL_SPATIAL=14,
   VS_INTERLACEMETHOD_VDPAU_TEMPORAL_SPATIAL_HALF=15,
   VS_INTERLACEMETHOD_DEINTERLACE_HALF=16,
-  VS_INTERLACEMETHOD_DXVA_BOB = 17,
```

Comment:
there are gaps already - 10 is missing. This is just an enum, you're not even obliged to give them  values, only if you want to make sure they persist :-) This doesn't cause issues.